_includes/integrations/webcomponent-quick-start.md

```diff
@@ -0,0 +1,60 @@
+## TinyMCE Web Component quick start guide
+
+The [Official {{site.productname}} Web Component](https://github.com/tinymce/tinymce-webcomponent) integrates {{site.productname}} into Web Component projects.
+This procedure creates a HTML page containing a {{site.productname}} editor based on our [Basic example]({{site.baseurl}}/demo/basic-example/).
+
+### Procedure
+
+To add a {{site.productname}} editor to a web page using the {{site.productname}} Web Component:
+
+1. Add the `DOCTYPE` element to the target page, such as:
+
+    ```html
+    <!DOCTYPE html>
+    ```
+
+    The `DOCTYPE` element is required for the editor to function correctly.
+
+2. Add the following `meta` elements to the head of page:
+
+    ```html
+    <head>
+      <meta charset="utf-8"/>
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    ```
+
+    The second `meta` element is required for the editor to function correctly on mobile devices.
```

Comment:
I'd vote we either [link out to mdn](https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag) or don't bother explaining what the meta is for.

Reply:
Seems like this is general information that applies everywhere and isn't specific to web components.

Reply:
yep. We need to assume that this is the first page they will land on and they may be new users, hence the walk-through

Reply:
I'll add the link next week.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1659/commits/35809236642e6faf2771966c7af9ba7c0d29d940